### DISCUSSION TITLE
Merging Mitosis with Qwik

### DISCUSSION THREAD

**Discussion by tirrth** [BuilderIO/jsx-lite#922] [Ideas]
It would sound just pretty much clear from the title itself of merging mitosis with qwik. So, one can make components in qwik and the same components can also be transferred into other frameworks. Qwik is already a framework of re-imagination. So, why not just add more workflows into it and make qwik a global coding syntax that everyone would want to use because of its robustness and generalization. Just a qwik idea 😀

**Comment by samijaber** [comment]
"Merging" isn't really the right word here: what you're suggesting is adding another Mitosis syntax parser that can consume Qwik component syntax.

> So, one can make components in qwik and the same components can also be transferred into other frameworks

Yeah, if there's enough demand/interest, we can add a Mitosis parser that parses Qwik-style JSX syntax and converts that to the Mitosis Intermediate Representation JSON. It's definitely something that anyone outside of the Builder team could take on too, like how @raymondmuller built the Svelte syntax parser and contributed it to the project.
